x/oracle/keeper/grpc_query_test.go

```diff
@@ -0,0 +1,132 @@
+package keeper_test
+
+import (
+	"context"
+	"math/rand"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/umee-network/umee/x/oracle/types"
+)
+
+func (s *IntegrationTestSuite) TestQuerier_ActiveExchangeRates() {
+	const (
+		exchangeRate = "umee"
+	)
+	app, ctx := s.app, s.ctx
+
+	app.OracleKeeper.SetExchangeRate(ctx, exchangeRate, sdk.OneDec())
```

Comment:
```suggestion

	s.app.OracleKeeper.SetExchangeRate(s.ctx, exchangeRate, sdk.OneDec())
```

Reply:
(Apply to rest of the tests too)

I was the source of this `app, ctx := s.app, s.ctx` line in the leverage tests, but it has since been removed upon review, so I'm propagating that decision to this file.